utils/script.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Script binding for function entry and exit
+ *
+ * Copyright (C) 2017, LG Electronics, Honggyu Kim <hong.gyu.kim@lge.com>
+ *
+ * Released under the GPL v2.
+ */
+#ifndef __UFTRACE_SCRIPT_H__
+#define __UFTRACE_SCRIPT_H__
+
+#include "libmcount/mcount.h"
+#include "utils/script-python.h"
+
+/* script type */
+enum script_type_t {
+	SCRIPT_UNKNOWN = 0,
+	SCRIPT_PYTHON
+};
+
+/* parameter information passed to script */
+struct script_info {
+	int tid;
+	int depth;
+	uint64_t timestamp;
+	uint64_t duration;	/* exit only */
+	unsigned long address;
+	char *symname;
+};
+
+extern char *script_str;
+
+typedef int (*script_uftrace_entry_fp)(struct script_info *scinfo);
+typedef int (*script_uftrace_exit_fp)(struct script_info *scinfo);
```

Comment:
I've made it '_t' but kept two difference function pointers for later update because I may have to separate it again.